### ISSUE TITLE
[QUEST][4.3.4] "Last Rites"

### ISSUE THREAD

**Issue by DepaMarco** [TrinityCore/TrinityCore#10836]
TrinityCore rev. e740332c0b68 2013-09-15 23:20:13 +0200 (4.3.4 branch)

When I click "I'm ready to fight", nothing happens :(
http://www.youtube.com/watch?v=D78bUE8vVD0

**Comment by Kodekc** [comment]
For wotlk too. Maybe this will help:
https://community.trinitycore.org/topic/1319-wip-questsborean-tundra-last-rites/

**Comment by Rushor** [comment]
no 4.x branch in mainrepo any longer

**Comment by Aokromes** [comment]
This bug is valid on 6.x?

**Comment by tkrokli** [comment]
Nope, this quest works as intended, just finished testing it on rev. 96e5d9cd68dc 2016-06-14 19:01:30 +0200 (6.x branch) (Win64, Release) / TDB 6.04 + updates up to and including `2016_06_14_00_world.sql`.

I only safeguarded myself by using some OP gear on a level 75 character and a powerful spell, although I don't think that spell was needed either. Thassarian correctly started the event as soon as I clicked on the gossip option "Let's do this, Thassarian.  It's now or never." and then spawned NPC Prince Valanar `<The San'layn>` + the extra NPCs there.
